docs/modules/ROOT/examples/io/quarkiverse/langchain4j/samples/RetrievalAugmentorExample.java

```diff
@@ -0,0 +1,35 @@
+package io.quarkiverse.langchain4j.samples;
+
+import java.util.function.Supplier;
+
+import jakarta.inject.Singleton;
+
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.rag.DefaultRetrievalAugmentor;
+import dev.langchain4j.rag.RetrievalAugmentor;
+import dev.langchain4j.rag.content.retriever.EmbeddingStoreContentRetriever;
+import io.quarkiverse.langchain4j.redis.RedisEmbeddingStore;
+
+@Singleton
+public class RetrievalAugmentorExample implements Supplier<RetrievalAugmentor> {
+
+    private final RetrievalAugmentor augmentor;
+
+    RetrievalAugmentorExample(RedisEmbeddingStore store, EmbeddingModel model) {
+        EmbeddingStoreContentRetriever contentRetriever = EmbeddingStoreContentRetriever.builder()
+                .embeddingModel(model)
+                .embeddingStore(store)
+                .maxResults(20)
```

Comment:
That's a lot of document - this will exceed context size. Can you change it to something more realistic like 3.